### ISSUE TITLE
Study spaces maps are low res

### ISSUE THREAD

**Issue by zipy124** [uclapi/ucl-assistant-app#131]
As far as i'm aware we get SVG's but they show extremely low res eve upon zooming in the app. the writing on them is un-readable.

**Comment by hueyy** [comment]
IIRC they're not true SVGs in that they embed a PNG with the colourful dots overlayed on top. We could solve this by vectorising our floor maps, which is something @anushks has attempted to do.

**Comment by anushks** [comment]
Hi yes it takes forever LOL

**Comment by zipy124** [comment]
ahh, this would explain it i guess

**Comment by ChristopherHammond13** [comment]
Yeah these maps come from the OccupEye API. I'm not sure if there's a way to get higher resolution ones, but I can investigate this

**Comment by Feverfew** [comment]
We can use one of these solutions: https://stackoverflow.com/questions/1861382/how-to-convert-a-png-image-to-a-svg
First two images are free for vectormagic so we can see if it's worth paying for (or asking OccupEye to pay for it instead of us, or learn how to design an SVG). 

Others are programmatic which is nice (i.e. might be able to do this automatically API side).

**Comment by Feverfew** [comment]
Via `convert map.png -colorspace gray -threshold 94% map.pbm && potrace map.pgm -s -a 0` on Ubuntu 20.04 I managed to get a not too bad conversion. Can't upload on GitHub unfortunately, but would appreciate if someone else could tinker to see what's the best possible combination here.

**Comment by hueyy** [comment]
`convert map.png -colorspace gray -threshold 94% map.pbm && potrace map.pbm -s -a 0` produces this:

![map1](https://gist.githubusercontent.com/hueyy/9397ea6e76ddf9b6e803af2e0c63702a/raw/f38c2ef7389fddc11fa2672c35ab376382c42797/map1.svg)

I fiddled around a bit, but didn't manage to produce anything substantially better. 

[pypotrace](https://pythonhosted.org/pypotrace/index.html) seems like an easy way to do this programmatically, perhaps just before we cache the images in redis. 

vectormagick appears to produce similar results, so might not be worth paying for
![image](https://user-images.githubusercontent.com/6523121/82851038-11dafe80-9f31-11ea-973b-a4fb33a31056.png)
